src/QuickMarcEditor/QuickMarcEditorContainer.js

```diff
@@ -0,0 +1,77 @@
+import React, {
+  useEffect,
+  useState,
+  useCallback,
+} from 'react';
+import PropTypes from 'prop-types';
+import { withRouter } from 'react-router';
+import ReactRouterPropTypes from 'react-router-prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { LoadingView } from '@folio/stripes/components';
+import { baseManifest } from '@folio/stripes-acq-components';
+
+import { INVENTORY_INSTANCE_API } from '../common/constants';
+
+import QuickMarcEditor from './QuickMarcEditor';
+
+const QuickMarcEditorContainer = ({ mutator, match, onClose }) => {
+  const instanceId = match.params.instanceId;
+
+  const [instance, setInstance] = useState();
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    mutator.quickMarcEditInstance.GET()
+      .then(instanceResponse => {
+        setInstance(instanceResponse);
```

Comment:
`.then(setInstance)`

Reply:
in next PR it will have one more setState, so no reason to do `.then(setInstance)`